docs/en/rules/module.md

```diff
@@ -616,6 +616,7 @@ Name | Synopsis | Severity | Level
 
 Name | Synopsis | Severity | Level
 ---- | -------- | -------- | -----
+[Azure.ACR.ReplicaLocation](Azure.ACR.ReplicaLocation.md) | Container registry replica locations should be within allowed regions. | Important | Error
```

Comment:
This file is auto-generated. You don't need to update this file. Please remove it from the PR.